user-guide/Advanced_Modules/DOM/tutorials/DOM_Remove_Enum_Entry.md

```diff
@@ -0,0 +1,384 @@
+---
+uid: DOM_Remove_Enum_Entry
+---
+
+# Removing an enum entry from a GenericEnumFieldDescriptor
+
+In this tutorial, you will learn to soft-delete and permanently delete an enum entry from a GenericEnumFieldDescriptor.
+
+> [!NOTE]
+> This tutorial uses DataMiner version 10.3.x.
+
+## Prerequisites
+
+- DataMiner version 10.3.10 or higher
+
+- A DataMiner System with an [indexing database](xref:Indexing_Database)
+
```

Comment:
```suggestion

- Basic knowledge of DataMiner Object Models (DOM)
```

Reply:
In this tutorial, we assume the readers will know what to do with the DOM setup below and the other provided code excerpts. Should we, in that case, not add that a basic knowledge of DOM is a requirement to follow this tutorial?